app/views/conditions/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<table>
```

Comment:
This is great, but we may need to tailor it back depending on which of the rows the user stories call for.